### PR TITLE
Updating CallRecords API documentation - New fields added

### PR DESCRIPTION
**Pull request by Datar89**
We're including 8 new properties to the current response. We're not changing any behavior as part of these changes. The new properties that you will find are:

- New property for microsoft.graph.callRecords.session
    - isTest: Flag to indicate if the session corresponded to a session within test call.

- New properties for microsoft.graph.callRecords.participantEndpoint
    - name: Name of the device used by the media endpoint.
    - cpuName: CPU name used by the media endpoint.
    - cpuCoresCount: CPU cores count for the device used by the media endpoint.
    - cpuProcessorSpeedInMhz: CPU processor speed for the device used by the media endpoint.

- New properties for microsoft.graph.callRecords.mediaStream
    - rmsFreezeDuration: Root mean square for the received freeze duration related with the video stream.
    - averageFreezeDuration: Average for the received freeze duration related with video stream.
    - isAudioForwardErrorCorrectionUsed: Flag to indicate if the forward error correction (FEC) was used at some point in the call session.

**As part of this PR, we're updating** the examples for our API and definitions of the entities where the new fields were defined.

----------------------------------------------------------------

API Review: [Pull request 13002: 16271-adding-p1-fields-to-call-records-api - Repos (visualstudio.com)](https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/13002?path=/reviews/3806-ic3-data-platform-call-records-api-schema-extension/api.md&discussionId=156383)

CSDL File: [Pull request 7977927: [Beta] Adding new approved fields for CallRecords API - Repos (azure.com)](https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/7977927)

Roadmap item: Feature 16881 https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/16881

### CODE REVIEW

**Comment by Lauragra** [comment]
@Datar89 , please include a changelog record to call out the addition of the new properties. Thanks.

File: api-reference/beta/resources/callrecords-mediastream.md [hunk 1]
```diff
@@ -41,13 +43,13 @@ Represents information about a media stream between two endpoints in a call.
 |maxRoundTripTime|Duration|Maximum network propagation round-trip time computed as specified in [RFC 3550][], denoted in [ISO 8601][] format. For example, 1 second is denoted as `'PT1S'`, where 'P' is the duration designator, 'T' is the time designator, and 'S' is the second designator.|
 |packetUtilization|Int64|Packet count for the stream.|
 |postForwardErrorCorrectionPacketLossRate|Double|Packet loss rate after FEC has been applied aggregated across all video streams and codecs.|
+|rmsFreezeDuration|Duration|Root mean square for the received freeze duration related with video stream.|
```
Comment (GageAmes) [inline api-reference/beta/resources/callrecords-mediastream.md:46]: It appears `rmsFreezeDuration` may have been missed in the JSON representation below: https://github.com/microsoftgraph/microsoft-graph-docs/blob/13cfa77476bc58d4e2548a07fd2d5a2d2ea033b7/api-reference/beta/resources/callrecords-mediastream.md?plain=1#L91-L92

File: api-reference/beta/resources/callrecords-participantendpoint.md [hunk 2]
```diff
@@ -39,9 +43,11 @@ The following is a JSON representation of the resource.
 
 ```json
 {
-  "userAgent": {"@odata.type": "microsoft.graph.callRecords.userAgent"},
-  "feedback": {"@odata.type": "microsoft.graph.callRecords.userFeedback"},
-  "identity": {"@odata.type": "microsoft.graph.identitySet"}
+    "userAgent": {"@odata.type": "microsoft.graph.callRecords.userAgent"},"feedback": {"@odata.type": "microsoft.graph.callRecords.userFeedback"},"name": "String",
```
Comment (GageAmes) [inline api-reference/beta/resources/callrecords-participantendpoint.md:46]: Please place the `feedback` and `name` properties on separate lines

**Review by GageAmes — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/callrecords-participantendpoint.md (hunk 2 above)
Comment (Datar89) [reply api-reference/beta/resources/callrecords-participantendpoint.md]: Done @GageAmes. Thanks!

**Comment by Danielabom** [comment]
Contacted author to get info.

**Review by mcm223 — APPROVED** [review]

File: api-reference/beta/resources/callrecords-session.md [hunk 3]
```diff
@@ -30,6 +30,7 @@ Represents a user-user communication or a user-meeting communication in the case
 |endDateTime|DateTimeOffset|UTC time when the last user left the session. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`|
 |failureInfo|[microsoft.graph.callRecords.failureInfo](callrecords-failureinfo.md)|Failure information associated with the session if the session failed.|
 |id|string|Unique identifier for the session. Read-only.|
+|isTest|Boolean|True if session correspond to a test.|
```
Comment (Danielabom) [inline api-reference/beta/resources/callrecords-session.md:33]: Suggested description: Specifies if the session is a test.
And if there's a default value,  The default value is _______.

File: api-reference/beta/resources/callrecords-mediastream.md [hunk 4]
```diff
@@ -23,6 +23,7 @@ Represents information about a media stream between two endpoints in a call.
 |averageAudioDegradation|Double|Average Network Mean Opinion Score degradation for stream. Represents how much the network loss and jitter has impacted the quality of received audio.|
 |averageAudioNetworkJitter|Duration|Average jitter for the stream computed as specified in [RFC 3550][], denoted in [ISO 8601][] format. For example, 1 second is denoted as `'PT1S'`, where 'P' is the duration designator, 'T' is the time designator, and 'S' is the second designator.|
 |averageBandwidthEstimate|Int64|Average estimated bandwidth available between two endpoints in bits per second.|
+|averageFreezeDuration|Duration|Average for the received freeze duration related with video stream.|
```
Comment (Danielabom) [inline api-reference/beta/resources/callrecords-mediastream.md:26]:
```suggestion
|averageFreezeDuration|Duration|Average of the received freeze duration related to the video stream.|
```

File: api-reference/beta/resources/callrecords-mediastream.md [hunk 5]
```diff
@@ -32,6 +33,7 @@ Represents information about a media stream between two endpoints in a call.
 |averageVideoFrameRate|Double|Average frames per second received for a video stream, computed over the duration of the session.|
 |averageVideoPacketLossRate|Double|Average fraction of packets lost, as specified in [RFC 3550][], computed over the duration of the session.|
 |endDateTime|DateTimeOffset|UTC time when the stream ended. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`|
+|isAudioForwardErrorCorrectionUsed|Boolean|True if the forward error correction (FEC) was used at some point during session, false otherwise. Null if no information was registered.|
```
Comment (Danielabom) [inline api-reference/beta/resources/callrecords-mediastream.md:36]:
```suggestion
|isAudioForwardErrorCorrectionUsed|Boolean|Indicates if the forward error correction (FEC) was used at some point during the session. Null if no information was registered.|
```

File: api-reference/beta/resources/callrecords-mediastream.md [hunk 6]
```diff
@@ -32,6 +33,7 @@ Represents information about a media stream between two endpoints in a call.
 |averageVideoFrameRate|Double|Average frames per second received for a video stream, computed over the duration of the session.|
 |averageVideoPacketLossRate|Double|Average fraction of packets lost, as specified in [RFC 3550][], computed over the duration of the session.|
 |endDateTime|DateTimeOffset|UTC time when the stream ended. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`|
+|isAudioForwardErrorCorrectionUsed|Boolean|Indicates if the forward error correction (FEC) was used at some point during the session. Null if no information was registered.|
```
Comment (Danielabom) [inline api-reference/beta/resources/callrecords-mediastream.md:36]: @Datar89 does the property have a default value?

File: api-reference/beta/resources/callrecords-mediastream.md (hunk 1 above)
Comment (Danielabom) [inline api-reference/beta/resources/callrecords-mediastream.md:46]:
```suggestion
|rmsFreezeDuration|Duration|Root mean square of the received freeze duration related to the video stream.|
```

**Review by Danielabom — CHANGES_REQUESTED** [review]
@Datar89, thank you for working on this PR. I added comments; please review them. Thanks!

File: api-reference/beta/resources/callrecords-session.md [hunk 7]
```diff
@@ -30,6 +30,7 @@ Represents a user-user communication or a user-meeting communication in the case
 |endDateTime|DateTimeOffset|UTC time when the last user left the session. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`|
 |failureInfo|[microsoft.graph.callRecords.failureInfo](callrecords-failureinfo.md)|Failure information associated with the session if the session failed.|
 |id|string|Unique identifier for the session. Read-only.|
+|isTest|Boolean|True if session correspond to a test otherwise false.|
```
Comment (Danielabom) [inline api-reference/beta/resources/callrecords-session.md:33]:
```suggestion
|isTest|Boolean|Specifies if the session is a test.|
```

File: api-reference/beta/resources/callrecords-mediastream.md [hunk 8]
```diff
@@ -32,6 +33,7 @@ Represents information about a media stream between two endpoints in a call.
 |averageVideoFrameRate|Double|Average frames per second received for a video stream, computed over the duration of the session.|
 |averageVideoPacketLossRate|Double|Average fraction of packets lost, as specified in [RFC 3550][], computed over the duration of the session.|
 |endDateTime|DateTimeOffset|UTC time when the stream ended. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`|
+|isAudioForwardErrorCorrectionUsed|Boolean|True if the forward error correction (FEC) was used at some point during session, false otherwise. The default value is null in the case that no information was registered.|
```
Comment (Danielabom) [inline api-reference/beta/resources/callrecords-mediastream.md:36]:
```suggestion
|isAudioForwardErrorCorrectionUsed|Boolean|Indicates if the forward error correction (FEC) was used at some point during the session. The default value is null.|
```

**Review by Danielabom — APPROVED** [review]
Thank you for working on this PR @Datar89. PR approved.

**Pull request merged**